common/CMakeLists.txt

```diff
@@ -283,6 +283,7 @@ target_link_libraries(common_dbus PUBLIC
   common_log
   common_error
   common_events
+  common_processes
```

Comment:
Shouldn't it be the test depending on this, and not the library?

Reply:
Yup! Another good catch!